### ISSUE TITLE
Playlist downloading breaks on error.

### ISSUE THREAD

**Issue by b4skyx** [deepjyoti30/ytmdl#120]
Issue: Playlist downloading breaks when metadata of a song is not found. 
Error Displayed: No Data Found when looking for the playlist metadata.
Expected Behavior: The metadata merge should be skipped and the playlist next item should be queued for serarch/download.
 

Operating System: Archlinux
```
> ytmdl --version                                                                                           
2020.07.26
```
Config: Default

Playlist: PLwzxNJxD6BZy0iNETTxbCYUy22ujXy4pg
Url: https://www.youtube.com/playlist?list=PLwzxNJxD6BZy0iNETTxbCYUy22ujXy4pg

**Comment by b4skyx** [comment]
![image](https://user-images.githubusercontent.com/55960554/99547229-4fe74580-29dd-11eb-8aee-446fbb9f5489.png)
Screenshot of the above behavior reproduced.

**Comment by deepjyoti30** [comment]
@b4skyx  Thanks for reporting the issue.

The bug has been fixed in 347e9229c3

I would suggest you to install the latest commit from git and install either manually or go for the `ytmdl-git` package from AUR. This is because the latest commits have one specific change for the playlist functionality, that is, before searching for the song, it asks the user if to search the song with the title of the video or if the user would like to enter a different name,

When I tried to reproduce your issue, I just entered a different name for the song manually and the error of `No data found` did not occur.

Otherwise what you faced was also a bug. `ignore-errors` was not being considered when no data was found for the given song and thus it was exitting.

You can report back here if the issue still persists.

**Comment by b4skyx** [comment]
Made the commit changes to the source manually and it's working as expected. Working as it should, thank you!
